package.json

```diff
@@ -12,14 +12,20 @@
   "license": "MIT",
   "devDependencies": {
     "grunt": "0.4.5",
+    "grunt-cli": "^1.2.0",
     "grunt-contrib-connect": "0.5.0",
     "grunt-contrib-cssmin": "0.7.0",
     "grunt-contrib-watch": "0.5.3",
     "grunt-sass": "1.1.0",
     "grunt-swig-templates": "0.1.4",
     "grunt-version": "0.3.3",
     "jit-grunt": "0.9.1",
+    "load-grunt-tasks": "0.2.0",
     "node-bourbon": "4.2.3",
-    "load-grunt-tasks": "0.2.0"
+    "npm install --save-dev bower"
```

Comment:
`npm install --save-dev bower` should be run on the command line, and `bower` will be automatically added to this file